protocol-tests/pom.xml

```diff
@@ -72,6 +72,48 @@
       <version>1.7.36</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-exporter-logging</artifactId>
+      <version>1.11.0</version> <!-- {x-version-update;io.opentelemetry:opentelemetry-exporter-logging;external_dependency} -->
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-security-keyvault-secrets</artifactId>
+      <version>4.3.8</version> <!-- {x-version-update;com.azure:azure-security-keyvault-secrets;dependency} -->
```

Comment:
remove `<!-- {x-version-update;io.opentelemetry:opentelemetry-exporter-logging;external_dependency} -->`

and why keyvault?